contrib/ansible/Getting-Started.md

```diff
@@ -0,0 +1,156 @@
+
+
+# Kubernetes Cluster with Containerd and Cri-Containerd
+
+This document provides the steps to bring up a Kubernetes cluster using ansible and kubeadmusing ansible and kubeadm tools. 
+
+### Prerequisites:
+- **OS**: Ubuntu 16.04 (will be updated with additional distros after testing)
+- **Python**: 2.7+
+- **Ansible**: 2.3+
+
+## Step 0:
+-  Install Ansible on any one of the nodes in the candidate cluster or an external machine which will be used to provision the cluster. The install instructions can be found [here](http://docs.ansible.com/ansible/latest/intro_installation.html). The playbooks in the repo need ansible version 2.3 or later.
+-  Create a hosts file and include the Routable IPs of the nodes that need to be provisioned by ansible.
+```
```

Comment:
shell or console?

Reply:
shell is a console..

Reply:
I mean we should tag what the language is so that markdown will show it prettier.

Reply:
@abhinandanpb Sorry, I found that we should use console. Or else markdown will highlight some key words `in`, `for` etc.